components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,121 @@
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+}
+
+.denhaag-breadcrumb__item,
+.denhaag-breadcrumb__link {
+  align-items: center;
+  display: flex;
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child) {
+  --denhaag-breadcrumb-background-color: var(--denhaag-breadcrumb-dots-background-color, #f1f1f1);
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child):hover {
+  --denhaag-breadcrumb-background-color: var(--denhaag-breadcrumb-dots-hover-background-color, #d2d2d2);
+}
+
+.denhaag-breadcrumb__link {
+  color: var(--denhaag-breadcrumb-link-color, #094b81);
+  padding-block-start: var(--denhaag-breadcrumb-spacing, 8px);
+  padding-block-end: var(--denhaag-breadcrumb-spacing, 8px);
+  padding-inline-start: var(--denhaag-breadcrumb-spacing, 8px);
+  padding-inline-end: var(--denhaag-breadcrumb-spacing, 8px);
+  pointer-events: none;
+  position: relative;
+}
+
+.denhaag-breadcrumb__link:hover,
+.denhaag-breadcrumb__link--hover {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-link-hover-color, #094b81);
+}
+
+.denhaag-breadcrumb__link:focus,
+.denhaag-breadcrumb__link--focus {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-link-focus-color, #1261a3);
+  outline: var(--denhaag-breadcrumb-link-focus-outline, var(--denhaag-link-focus-outline));
+}
+
+.denhaag-breadcrumb__link--current,
+.denhaag-breadcrumb__link.denhaag-link--current,
+.denhaag-breadcrumb__item:last-child > .denhaag-breadcrumb__link {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-current-color, var(--denhaag-breadcrumb-color, #4b4b4b));
+  text-decoration: none;
+}
+
+.denhaag-breadcrumb__link:not(.denhaag-breadcrumb__link--current):not(.denhaag-breadcrumb__link.denhaag-link--current):not(:last-child
+    > .denhaag-breadcrumb__link)
+  > :first-child {
+  pointer-events: bounding-box;
+}
+
+/* SVG for the separator */
+.denhaag-breadcrumb__link > .denhaag-breadcrumb__text + .denhaag-icon {
+  color: initial;
+  font-size: inherit;
+  margin-inline-start: var(--denhaag-breadcrumb-spacing, 8px);
+  margin-inline-end: calc(-1 * var(--denhaag-breadcrumb-spacing, 8px));
+}
+
+[dir="rtl"] .denhaag-breadcrumb__link > .denhaag-breadcrumb__text + .denhaag-icon {
+  transform: scaleX(-1);
+}
+
+.denhaag-breadcrumb__text {
+  background: var(--denhaag-breadcrumb-background-color);
+  overflow: hidden;
+  position: relative;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.denhaag-breadcrumb__link > .denhaag-icon + .denhaag-breadcrumb__text {
+  /* Copied from Bootstrap .sr-only */
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  margin-inline-start: -1px;
+  margin-inline-end: -1px;
+  margin-block-start: -1px;
+  margin-block-end: -1px;
+  overflow: hidden;
+  padding-block-start: 0;
+  padding-block-end: 0;
+  padding-inline-start: 0;
+  padding-inline-end: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+}
```

Comment:
Similar to in Pagination, I wonder if we can avoid this class by proper usage of aria-attributes.

Reply:
Replaced with Aria